src/main/java/net/fabricmc/tinyremapper/AsmClassRemapper.java

```diff
@@ -277,12 +279,15 @@ public void visitInvokeDynamicInsn(String name, String descriptor, Handle bootst
 					bootstrapMethodArguments);
 		}
 
+		private static final Set<String> ALLOWED_INDY_BSM = Stream.of(System.getProperty("net.fabricmc.tinyremapper.knownindybsm", "").split(",")).collect(Collectors.toSet());
```

Comment:
This should be set via the CLI and via the TinyRemapper.Builder. See how `forcePropagation` is set.